### PR TITLE
breaking: remove Svelte 3 support

### LINKED ISSUE #30492: breaking: remove Svelte@3 support
[Svelte 5](https://www.npmjs.com/package/svelte/v/5.0.0) released 10 days ago on `npm` as an official release. This means Svelte 3 is two major versions behind (last released a year ago) and we should no longer support it

### Other

[Cypress 14 breaking changes](https://github.com/cypress-io/cypress/issues/29925)

### PR DESCRIPTION
**Pull request by mschile**
- Closes https://github.com/cypress-io/cypress/issues/30492

### Additional details

Removes support for Svelte version 3.

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? https://github.com/cypress-io/cypress-documentation/pull/5999
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: npm/svelte/rollup.config.mjs [hunk 1]
```diff
@@ -1,3 +1,3 @@
 import { createEntries } from '@cypress/mount-utils/create-rollup-entry.mjs'
 
-export default createEntries({ formats: ['es', 'cjs'], input: 'src/index.ts' })
+export default createEntries({ formats: ['es', 'cjs'], input: 'src/index.ts', dtsOptions: { respectExternal: false } })
```
Comment (AtofStryker) [inline npm/svelte/rollup.config.mjs:4]: can you link the git issue here as to why we needed to do this again?

File: cli/CHANGELOG.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ _Released 12/3/2024 (PENDING)_
 - Cypress Component Testing no longer supports `Vue` version 2. Addresses [#30295](https://github.com/cypress-io/cypress/issues/30295).
 - Cypress Component Testing no longer supports `Next.js` versions 10, 11, 12, and 13. Addresses [#29583](https://github.com/cypress-io/cypress/issues/29583).
 - Cypress Component Testing no longer supports `Angular` versions 13, 14, 15, and 16. The minimum supported version is now `17.2.0` in order to fully support Angular [signals](https://angular.dev/guide/signals). Addresses [#29582](https://github.com/cypress-io/cypress/issues/29582). Addressed in [#30539](https://github.com/cypress-io/cypress/pull/30539).
+- Cypress Component Testing no longer support `Svelte` version 3. Addresses [#30492](https://github.com/cypress-io/cypress/issues/30492)
```
Comment (jennifer-shehane) [inline cli/CHANGELOG.md]:
```suggestion
- Cypress Component Testing no longer supports `Svelte` version 3. Addresses [#30492](https://github.com/cypress-io/cypress/issues/30492)
```

**Review by jennifer-shehane — APPROVED** [review]

File: cli/CHANGELOG.md (hunk 2 above)
Comment (AtofStryker) [reply cli/CHANGELOG.md]: I'd add a period to the end as well

**Review by AtofStryker — APPROVED** [review]

**Comment by AtofStryker** [comment]
can you link the docs PR as well once it's ready for review?

**Comment by mschile** [comment]
> can you link the docs PR as well once it's ready for review?

https://github.com/cypress-io/cypress-documentation/pull/5999

File: npm/svelte/rollup.config.mjs (hunk 1 above)
Comment (mschile) [reply npm/svelte/rollup.config.mjs:4]: updated [60dd249](https://github.com/cypress-io/cypress/pull/30572/commits/60dd24993271dac64fbffc95702152a10adfb54b)

File: cli/CHANGELOG.md (hunk 2 above)
Comment (mschile) [reply cli/CHANGELOG.md]: updated [60dd249](https://github.com/cypress-io/cypress/pull/30572/commits/60dd24993271dac64fbffc95702152a10adfb54b)

**Pull request merged**